recipes/protobufc/LICENSE.txt

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2008-2023, Dave Benson and the protobuf-c authors.
```

Comment:
This file is not needed

Reply:
updated as suggested in c48273d8a0c4fb9955ed0ffe76b612b7906d724b